src/hooks/v2/transactor/LaunchProjectWithNftsTx.ts

```diff
@@ -40,19 +41,24 @@ function getJBDeployTieredNFTRewardDataSourceData({
   ownerAddress: string
   directory: string
 }) {
-  const tiersArg: ContractNftRewardTier[] = Object.keys(nftRewards).map(cid => {
```

Comment:
might be worth keeping the type. wdyt?

Reply:
It's a different type though. What we pass to the contract and what the contract gives back to us is slightly different now. I didn't think it was worth making a specific type for what we pass